dusty/source.py

```diff
@@ -158,19 +164,19 @@ def local_is_up_to_date(self):
 
     def update_local_repo(self, force=False):
         """Given a remote path (e.g. github.com/gamechanger/gclib), pull the latest
-        commits from master to bring the local copy up to date."""
+        commits from the active branch to bring the local copy up to date."""
         self.ensure_local_repo()
 
         logging.info('Updating local repo {}'.format(self.remote_path))
 
         managed_repo = git.Repo(self.managed_path)
         with git_error_handling():
-            managed_repo.remote().pull('master')
+            managed_repo.remote().pull(self.branch)
```

Comment:
Do we need an extra check here now to make sure this branch is checked out before pulling? I'm concerned about getting in a dirty state if the local copy is currently on master, then I change to a different branch in Dusty and hit this code path.